src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -1354,13 +1354,13 @@ public void fetchParticipationsWithSubmissionsAndResultsForCourses(List<Course>
     }
 
     /**
-     * DELETE /courses/:courseId/instructors/:instructorLogin : Remove the given user from the instructors of the course so that the instructor cannot access the course administration any more
+     * DELETE courses/:courseId/instructors/:instructorLogin : Remove the given user from the instructors of the course so that the instructor cannot access the course administration any more
```

Comment:
```suggestion
     * DELETE /courses/:courseId/instructors/:instructorLogin : Remove the given user from the instructors of the course so that the instructor cannot access the course administration any more
```